packages/aws-cdk-lib/aws-ses/lib/configuration-set.ts

```diff
@@ -73,6 +73,32 @@ export interface ConfigurationSetProps {
    * @default - use the default awstrack.me domain
    */
   readonly customTrackingRedirectDomain?: string;
+
+  /**
+   * The Virtual Deliverability Manager (VDM) options that apply to the configuration set
+   *
+   * @default - use account level settings
```

Comment:
We should probably indicate what the default values are first, and mention that this can be overridden by account level settings

Reply:
Thank you for the review.
I've stated that "default" indicates the absence of settings at the Configuration level, and in such cases, settings at the account level will be applied.

Reply:
Sorry, I should have made myself clearer. I think it's more important to tell the users that by default, VDM options are disabled, unless there are account level settings. Your current documentation only states that if they happen to have account level settings, those will be used. I would expect most of the CDK users will not be using account level settings.

Reply:
I understand. Sorry for the lack of explanation.

However, my understanding is that the configuration set level settings need to be used together with the account level settings. (Deploying with only the configuration set level settings enabled is possible, but VDM itself cannot be used until the account level settings are enabled.)

That's why I tried to include the above as a Note in the README, but I would like your opinion.
I'm a little unsure about how it should be written...

Reply:
Oh, got it. The AWS documentation doesn't really make it explicit that they need to be enabled account wide first:
> Virtual Deliverability Manager options are also provided at the configuration set level so you can define custom settings for how a configuration set will use engagement tracking and optimized shared delivery by overriding how they’ve been defined in Virtual Deliverability Manager.

I would really emphasize it in the TSDoc as well, the README seems to have it covered

Reply:
@nmussy 
Thank you. I have added a note to the README and docs that when setting Account Level settings with CDK, VdmAttributes should be used.